Core/App.Web/Options/ControllerOptions.cs

```diff
@@ -30,4 +38,20 @@ public IControllerOptions UseViews()
         UsingViews = true;
         return this;
     }
+
+    public IControllerOptions AddRequestPipeLineFilter(Type filter)
```

Comment:
Mag dit ook zijn: 
public IControllerOptions AddRequestPipeLineFilter<TFilter>() where TFilter is IFilterMetadata.
of
public IControllerOptions AddRequestPipeLineFilter(params Type filter[])

Of als overloads beide opties? Ik denk dat het iets developer friendly is.